include/clap/ext/draft/location.h

```diff
@@ -0,0 +1,72 @@
+#pragma once
+
+#include "../../plugin.h"
+#include "../../color.h"
+
+// This extension allows a host to tell the plugin more about its position
+// within a project or session.
+
+static CLAP_CONSTEXPR const char CLAP_EXT_LOCATION[] = "clap.location/1";
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+enum {
+   // Represents a document/project/session.
+   CLAP_PLUGIN_LOCATION_PROJECT,
```

Comment:
Elsewhere in CLAP we use explicit values for enums. So like `CLAP_PLUGIN_LCOATION_PROJECT=1` and so on.

Reply:
Yes.